packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -207,6 +208,14 @@ module.exports = {
     ]
   },
   plugins: [
+    // Lint our CSS
+    new StyleLintPlugin({
+      // @remove-on-eject-begin
+      configFile: path.join(__dirname, '../.stylelintrc'),
+      // @remove-on-eject-end
+      context: paths.appSrc,
+      files: ['**/*.css'],
```

Comment:
one thing to consider is that `stylelint` can handle `.sass`/`.scss` files, and the [default glob for stylelint-webpack-plugin supports both](https://github.com/JaKXz/stylelint-webpack-plugin/blob/master/lib/constants.js#L4) so you could drop this line.